src/util/llvm_lib.cpp

```diff
@@ -0,0 +1,28 @@
+/* Copyright © 2017 Apple Inc. All rights reserved.
```

Comment:
I would consider adding a new directory `src/stack_trace` or something like that, and moving the llvm_lib and stack_trace stuff there. It seems a bit large to add into the existing utils.